frontend/Store.js

```diff
@@ -288,13 +288,21 @@ class Store extends EventEmitter {
       this.emit('hover');
       this._bridge.send('highlight', id);
     } else if (this.hovered === id) {
-      this.hovered = null;
-      this.emit(id);
-      this.emit('hover');
-      this._bridge.send('hideHighlight');
+      this.hideHighlight();
     }
   }
 
+  hideHighlight() {
+    this._bridge.send('hideHighlight');
```

Comment:
oh yes! I totally did. No, nothing listens for hover at the moment. Would you rather I remove it?